### PR TITLE
feat: release Swift SDK to CocoaPods

### PR DESCRIPTION
**Pull request by brigonzalez**
## Description

- release Swift SDK to CocoaPods

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -43,3 +47,19 @@ jobs:
...
           github_token: ${{ secrets.SEMANTIC_RELEASE_PAT }}
           branch: ${{ github.ref }}
+
+  release-cocoapod:
+    name: Release CocoaPod
+    runs-on: macOS-latest
+    needs:
+      - tag-repo
+
+    if: ${{ needs.tag-repo.outputs.releaseType != '' }}
```
Comment (brigonzalez) [inline .github/workflows/release.yml:57]: 👀 hopefully the `needs` context is set here when it goes to check this condition. will need to keep an eye on this

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+    - master
```
Comment (koalaty-code) [inline .github/workflows/pull-request.yml:6]:
```suggestion
      - master
```
Nitpick, consistent spacing for arrays

File: .github/workflows/release.yml (hunk 1 above)
Comment (koalaty-code) [reply .github/workflows/release.yml:57]: Looks like this output never gets set if a new tag isn't made in `github-tag-action`. Do we need to account for `releaseType: undefined`?

**Review by koalaty-code — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (brigonzalez) [reply .github/workflows/release.yml:57]: I've had this check other places and haven't had an issue with `chore` commits. so I think we're good... surprisingly

**Review by koalaty-code — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (brigonzalez) [reply .github/workflows/release.yml:57]: my guess is that GH is defaulting that output to `''`

**Pull request merged**